packages/zipkin/src/tracer/index.js

```diff
@@ -155,6 +155,25 @@ class Tracer {
     });
   }
 
+  join(traceId) {
+    if (traceId === None) {
+      return new Some(this.createRootId());
```

Comment:
why would we return an option type here? would it not be easier to return a trace ID not wrapped?

Reply:
This is just to align to the same value type return below.  Hence, I try to reply both to your comment here.

The `join` method provided in tracer is intended to backfilling the span/trace created from upstream.  The `readHeader` method in HttpServer instrumentation return Some/None wrapping the spanId/traceId.  Hence, the input of traceId should normally be Some/None.

Then for below condition branch, possible original pass in traceId is returned or a childId wrapped in Some is returned.

Hence, to align input datatype or return value, I use Some to wrap it.

To be honest, I am not sure if there is any guideline on when Some/None should be used to wrap the raw value.  Please let me know if there is any and my comment above does not handle reasonably.